front/src/common/authorization/components/GrantsManager.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react';
+
+import { ChevronDown, ChevronUp } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+import type { Grant, PrivilegesByGrant, ResourceType } from 'common/api/mock/mockEditoastApi';
+import GrantsLabel from 'common/authorization/consts';
+import { capitalizeFirstLetter } from 'utils/strings';
+
+import GrantsManagerSubjects from './GrantsManagerSubjects';
+
+type GrantsManagerProps = {
+  resourceId: number;
+  resourceType: ResourceType;
+  userGrant: Grant;
+  privilegesByGrant?: PrivilegesByGrant;
+};
+
+const GrantsManager = ({
+  resourceId,
+  resourceType,
+  userGrant,
+  privilegesByGrant = {
+    READER: [],
+    WRITER: [],
+    OWNER: [],
+  },
+}: GrantsManagerProps) => {
+  const { t } = useTranslation('common/common');
+  const { t: tWithPrefix } = useTranslation('common/common', { keyPrefix: 'authorization' });
+  const [displayGrantSection, setDisplayGrantSection] = useState(false);
+
+  return (
+    <div
+      className="grant-manager"
+      role="button"
+      tabIndex={0}
+      // We need that to prevent the modal to close when clicking on this section
+      onClick={(e) => e.stopPropagation()}
+    >
+      <div className="grant-manager-header">
+        <span className="user-grant">
+          {tWithPrefix('yourGrant', {
+            // TODO: remove the fallback when the backend is ready, we won't have access to resources without grant
+            grant: capitalizeFirstLetter(tWithPrefix(`grants.${GrantsLabel[userGrant] || 'none'}`)),
```

Comment:
You can remove tWithPrefix and add the key here, it's easier to read
```suggestion
          {t('authorization.yourGrant', {
            // TODO: remove the fallback when the backend is ready, we won't have access to resources without grant
            grant: capitalizeFirstLetter(t(`authorization.grants.${GrantsLabel[userGrant] || 'none'}`)),
```